### PR TITLE
feat: Allow to skip app install if a newer/same app version is already installed

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Addresses https://github.com/appium/appium/issues/18279

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@KazuCocoa could you please help testing it on real devices?

**Comment by KazuCocoa** [comment]
Yes, of course, I will

**Review by KazuCocoa — APPROVED** [review]
Tested various version comparison cases on a real device. Bad semver case as well.

https://github.com/appium/appium-xcuitest-driver/pull/1514/commits/44a5c97229599fe4e9ff6b7448bf590aba4d2c3d had my thought as well

File: docs/capabilities.md [hunk 1]
```diff
@@ -17,14 +17,15 @@ Capability&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbs
...
+`appium:enforceAppInstall` | If set to `false` it will make xcuitest driver to verify whether the app version currently installed on the device under test is older than the one, which is provided as `appium:app` value. No app reinstall is going to happen if the candidate app has the same or older version number than the already installed copy of it. The version number used for comparison must be provided as [CFBundleVersion](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion) [Semantic Versioning](https://semver.org/)-compatible value in the application's Info.plist. No validation is performed by default, e.g. the provided app is always (re)installed, which could potentially slow down your test suites. | false
```
Comment (KazuCocoa) [inline docs/capabilities.md:28]: Ah.. maybe we should address that the installed app process could remain (as skill remove/install). So the user should make sure the app process termination by themselves

Comment (KazuCocoa) [reply docs/capabilities.md:28]: `forceAppLaunch` could help then

**Pull request merged**